package.json

```diff
@@ -43,5 +43,8 @@
     "**/*.{ts,js,tsx,scss,css,html,md}": [
       "yarn prettier --write"
     ]
+  },
+  "dependencies": {
+    "@types/node": "^16.11.4"
   }
```

Comment:
Why did this need to be added to the dependencies?

Reply:
+1

Reply:
@henry-li-06 I wasn't sure about this dependency and the lock file - these were added after we tried fixing my npm and yarn modules.

Reply:
I think this dependency can probably be deleted, `package-lock.json` should definitely be deleted though

Reply:
Sounds good, what about the edits to `yarn.lock`?